modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc

```diff
@@ -41,18 +41,37 @@ broker addresses:
 |===
 |*Value* |*Type* |*Description*
 
+|--admin-ports |strings |Redpanda Admin API ports to listen on; check help text for more information.
+
+|--any-port |- |Opt in for any (random) ports in all listeners.
+
+|--console-image |string |An arbitrary container Redpanda Console image to use (default `redpandadata/console:latest`).
+
+|--console-port |string |Redpanda console ports to listen on; check help text for more information (default `8080`).
+
+|--gateway |string |Gateway IP address for the subnet. Must be in the subnet address range (default `172.24.1.1`).
+
 |-h, --help |- |Help for start.
 
-|--image |string |An arbitrary container image to use (default `redpandadata/redpanda:v{full-version}`).
+|--image |string |An arbitrary container Redpanda image to use (default `redpandadata/redpanda:v{full-version}`).
```

Comment:
```suggestion
|--image |string |An arbitrary Redpanda container image to use (default `redpandadata/redpanda:v{full-version}`).
```
If you pass in a value for `--image` doesn't that mean that it's no longer arbitrary? How does the default work? Is it applied if the `--image` flag is not used at all?

Reply:
>If you pass in a value for --image doesn't that mean that it's no longer arbitrary?

If you pass in a value for --image, rpk will attempt to use that image. Otherwise, it will use the default.

> How does the default work? Is it applied if the --image flag is not used at all?

Correct.

Reply:
I added some parts on the main help text to help clarify this. Let me know if I should bring the content over to the flags section too.